### DISCUSSION TITLE
关于全象云低代码的一则公告

### DISCUSSION THREAD

**Discussion by hqc19907228** [quanxiang-cloud/quanxiang#41] [Announcements]
各位一直关注全象云低代码的朋友们大家好，今天要宣布一个很遗憾的消息：鉴于一些原因，全象云低代码的开发工作将会暂时停止。

这则消息来的太过突然，对于我们团队来说亦是如此。虽然项目已经停止，但有始也须有终，一些事情还需收尾。GitHub 上面的项目 [quanxiang-cloud](https://github.com/quanxiang-cloud) 会一直保留，大家如果有时间可以提交一些 commit。此外，我们将原本计划逐步开放的组件库进行全部的开源，下方表格列出了所有的开源组件、功能与地址，希望能够对大家有所帮助。

### 全象开源组件及功能列表

| 组件名称 | 组件功能 | 组件链接 |
| --- | --- | --- |
| app-center | 应用管理中心：应用基本信息及应用权限管理 |	https://github.com/quanxiang-cloud/appcenter |
| audit |	无	|
| dispatcher |	时间调度服务： 定时回掉指定任务接口 |	
| entrepot	| 任务管理中心：异步任务管理中心	| https://github.com/quanxiang-cloud/entrepot |
| fileserver |	文件服务：支持 aws s3 协议的对象存储上传与下载 |	https://github.com/quanxiang-cloud/fileserver |
| flow |	低代码流程引擎：低代码流程定义、低代码业务节点扩展和低代码其它业务整合 | https://github.com/quanxiang-cloud/flow |	
| form |	表单引擎：表单高级组件、以及 schema 的处理，与 structor 配合使用	| https://github.com/quanxiang-cloud/form |
| goalie |	权限管理：角色权限管理，RBAC 权限模型	|
| kms| 	密钥管理：平台密钥管理及签名验证，外部密钥代理及鉴权	| https://github.com/quanxiang-cloud/kms |
| message |	消息服务：消息内容管理，邮件、站内信分发 | https://github.com/quanxiang-cloud/message |
| nurturing |	已废弃	|
| organizations |	组织服务：人员部门等信息管理，内置人员扩展字段服务功能	| https://github.com/quanxiang-cloud/organizations |
| persona |	应用配置中心：应用个性化配置数据存储	| https://github.com/quanxiang-cloud/persona |
|polyapi |	API 管理：API 注册，API 文档管理，第三方 API 代理，API 编排，API 统一调用	| https://github.com/quanxiang-cloud/polyapi |
| polygate	| API 网关：token/signature 认证，透明代理	| https://github.com/quanxiang-cloud/polygate |
| process |	流程引擎内核：流程模型定义、流程调度和实例数据记录	| https://github.com/quanxiang-cloud/process |
| qxp-web-home | web 用户端服务	| https://github.com/quanxiang-cloud/qxp-web |
| qxp-web-nginx	| web 静态文件，后面要废弃 |
| qxp-web-portal |	web 管理端服务	|
| structor |	元数据管理：CURD 数据抽象，对接数据库管理	| https://github.com/quanxiang-cloud/structor |
| transaction |	已废弃	|
| warden |	认证服务：jwt 协议认证，生产管理 accesstoken，refreshtoken，支持第三方 jwt 协议 sso；|

除此之外，还有我们的博客版块，该部分内容全部是全象开发团队写作分享的一些技术干货，原计划在官网下一版本更新中增加，大家有兴趣可以点击 [Blog](https://github.com/quanxiang-cloud/website/tree/main/content/zh/blogs) 进行查看，也可以访问我们的公众号/[知乎号](https://www.zhihu.com/people/quan-xiang-yun-di-dai-ma/posts)（全象云低代码）查看历史内容。

最后，感谢各位开发者一直以来的不懈努力，感谢各位关注者一如既往的不离不弃。如果有缘，希望我们再见！

**Comment by rengeng1986** [comment]
膜拜大牛，再次感谢！

**Reply by hqc19907228** [reply]
以后可能也就在讨论区冒冒泡了，不过有空可以多来逛逛！~

**Comment by BarronLin** [comment]
怎么会这么突然呀，我一直关注全象云低代码的进展，我觉得全象云低代码是非常有价值的！

**Reply by hqc19907228** [reply]
可能老板觉得项目没啥营收，并且关注度不高吧？其实应该早点开个讨论区的，这样就能把潜水的兄弟们都炸出来了，至少显得一片欣欣向荣~    :）

**Reply by BarronLin** [reply]
看看宜搭，市场需求是很庞大的，低代码的价值也越来越得到市场肯定。按照全象云低代码的路线规划是覆盖了宜搭核心功能集，同时又提供了OpenAPI、Faas服务端定制能力，再依托青云的kubesphere容器云平台，全象云低代码就实现了前端低代码、服务端低代码、运维自动化的真企业生产级低代码平台。市场层面来看，全象云低代码可以很好适配中大型企业对低代码平台化和私有化部署的需求，特别对应宜搭不支持私有化部署，对小型企业或者个人也提供半免费全象云Saas产品服务，两方面营收足以支撑全象云团队活着了。

**Reply by kevinlee607** [reply]
项目还会继续下去，只是更新会很慢。希望大家持续关注。也希望大家多多提供commit。

**Reply by hqc19907228** [reply]
> 看看宜搭，市场需求是很庞大的，低代码的价值也越来越得到市场肯定。按照全象云低代码的路线规划是覆盖了宜搭核心功能集，同时又提供了OpenAPI、Faas服务端定制能力，再依托青云的kubesphere容器云平台，全象云低代码就实现了前端低代码、服务端低代码、运维自动化的真企业生产级低代码平台。市场层面来看，全象云低代码可以很好适配中大型企业对低代码平台化和私有化部署的需求，特别对应宜搭不支持私有化部署，对小型企业或者个人也提供半免费全象云Saas产品服务，两方面营收足以支撑全象云团队活着了。

看到这样的对全象的理解和反馈真的很开心了！目前还有 KevinLee 及一些同事在做维护，希望这是短暂的山重水复疑无路，期待未来的柳暗花明又一村。

**Reply by persistence** [reply]
低代码、无代码，很有市场的！希望能够继续做下去。

**Comment by linkagt** [comment]
低代码目前很多产品都还没看到有什么产品可以很好的满足市场的需求，to b的市场其实更重要的是业务、流程、还有规则引擎的业务化，开源的好处是验证市场化。而更觉得本项目如果本项目做的好，对于青云是未来的一个大引擎，毕竟任何企业是需要云计算资源的，本来还想测试和提些建议，突然停掉挺可惜的，从产品目前资料可看到的，还是觉得青云的开源项目都是良心项目，都是比较商业化质量比较高，用心去做的项目。

**Reply by kevinlee607** [reply]
可以试试，现在功能基本已经能够满足企业很多需求了，后续走纯社区运营，看有没有开源爱好者一起让社区活跃起来。

**Comment by linkagt** [comment]
我们这边愿意参与，但是得先看看熟悉了先，青云的开源产品真的都是中国开源产品的标杆，很多实际已经满足商业化的了，包括我们现在k8s集群管理用的也是青云的开源



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年6月27日(周一) 下午2:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [quanxiang-cloud/quanxiang] 关于全象云低代码的一则公告 (Discussion #41)




 
可以试试，现在功能基本已经能够满足企业很多需求了，后续走纯社区运营，看有没有开源爱好者一起让社区活跃起来。

**Reply by hqc19907228** [reply]
好啊，欢迎欢迎！现在真的是纯社区模式运作了，有兴趣参与的话可以先进 slack 交流群。

**Comment by qingpengchen2011** [comment]
太牛逼了~ 感谢

**Comment by persistence** [comment]
整体安装和部署，有手册或者文档吗？

**Reply by kevinlee607** [reply]
readme里有

**Comment by BigBaBei** [comment]
好厉害的项目，怎么就关闭了，不过谢谢各位大佬们的分享，学习下
